build.gradle

```diff
@@ -48,11 +48,20 @@ subprojects {
     strictCheck true
   }
 
+  configurations {
+    all {
+      exclude group: 'org.slf4j', module: 'slf4j-log4j12'
```

Comment:
Could you please add a comment with the first part of the PR description here? Would help with future maintenance.

Reply:
Done